ui/app/components/tool-actions-form.js

```diff
@@ -135,14 +134,11 @@ export default Component.extend(DEFAULTS, {
       this.reset();
     },
 
-    updateTtl(evt) {
-      const ttl = evt.enabled ? `${evt.seconds}s` : '30m';
+    updateTtl(ttl) {
       set(this, 'wrapTTL', ttl);
     },
 
-    codemirrorUpdated(val, codemirror) {
-      codemirror.performLint();
```

Comment:
Same, can codemirror be undefined or null?

Reply:
nope, it's always an object that's returned.